### DISCUSSION TITLE
ProtonVPN random or list of peer/locations to rotate?

### DISCUSSION THREAD

**Discussion by x0tester0x** [qdm12/gluetun#1942] [Q&A] [answered]
How can I configure gluetun (ProtonVPN) to select a peer/location randomly or from a list with WireGuard and/or OpenVPN?

**Comment by gmillerd** [comment, marked as answer]
openvpn you flood the openvpn.conf with endpoints (the line that starts with "remote ip.add.re.ss:port" is duplicated for each of your desired endpints and then the line "remote-random" is used to shuffle therein.

google "remote-random openvpn"

with wireguard this is not so easy, since each endpoint has a separate key and ugh ... you could have a separate dockersecrets file for each endpoint and then shuffle that, but that is not very robust.
